### ISSUE TITLE
Code editor isn't rendering in the test.

### ISSUE THREAD

**Issue by mmelko** [KaotoIO/kaoto-ui#973]
### Please describe the task that needs to be done

It appears that current code editor isn't really rendered. This might be actually an issue with Monaco Jest or missing some dependencies or mocks. 

This is current render of SourceCodeEditor component in the test:
```html
body>
  <div>
    <textarea />
  </div>
</body>
```

**Comment by Delawen** [comment]
@lordrip is this still happening? How can I check?

**Comment by lordrip** [comment]
Yes, it's still happening. This is because the `monaco` editor is a bit complex to run it through the JSDom environment.

We're [mocking the whole component here](https://github.com/KaotoIO/kaoto-ui/blob/d30e611a509a08d2268deb63ae7cf90b0ef282d3/src/__mocks__/reactCodeEditorMock.js)

In order to fix this, the idea is to split the buttons from the code editor itself, so we can at least check the interaction between Kaoto and the code editor.
